05-angularjs/keypoints/js/api.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Created by MaximilianoDaniel on 21/09/2015.
+ */
+var MovieApp = angular.module('MovieApp', ['ngRoute']);
+
+MovieApp.config(['$routeProvider', function ($routeProvider) {
+    $routeProvider.when('/index/:movieIndex/action/:action', {
+        templateUrl: 'template/abmMovie.html',
+        controller: 'MovieAbmCtrl'
+        });
+}]);
+
+MovieApp.controller('MovieListCtrl', ['$scope', 'Movies', function ($scope, Movies) {
+    Movies.clear();
+    $scope.movies = Movies.get();
+}]);
+
+MovieApp.service('Movies', function ($localStorage) {
```

Comment:
$localStorage should be injected right? It's good practice to use the coding style you've used for the MovieListCtrl